components/lvgl_esp32_drivers/lvgl_spi_conf.h

```diff
@@ -70,22 +70,19 @@ extern "C" {
 #define SPI_BUS_MAX_TRANSFER_SZ (DISP_BUF_SIZE * 2)
 #endif
 
-#if defined (CONFIG_LVGL_TFT_USE_CUSTOM_SPI_FREQUENCY)
-#define SPI_TFT_CLOCK_SPEED_HZ (CONFIG_LVGL_TFT_SPI_CUSTOM_FREQUENCY * 1000 * 1000)
+#if defined (CONFIG_LVGL_TFT_USE_CUSTOM_SPI_CLK_DIVIDER)
+#define SPI_TFT_CLOCK_SPEED_HZ (80 * 1000 * 1000 / CONFIG_LVGL_TFT_CUSTOM_SPI_CLK_DIVIDER)
```

Comment:
I think the multiplication operator have precedence over the divisor operator [1], but I think is better to use parenthesis for it.
```c
#define SPI_TFT_CLOCK_SPEED_HZ ((80 * 1000 * 1000) / CONFIG_LVGL_TFT_CUSTOM_SPI_CLK_DIVIDER)
```

[1] https://en.cppreference.com/w/c/language/operator_precedence